apps/journeys-admin/src/components/SingleJourney/CardOverview/CardOverview.spec.tsx

```diff
@@ -1,13 +1,30 @@
 import { render } from '@testing-library/react'
+import { renderWithApolloClient } from '../../../../test/testingLibrary'
+
 import CardOverview from './CardOverview'
+import { steps, oneStep } from './CardOverviewData'
 
 describe('CardOverview', () => {
-  it('should render button with the correct link', () => {
-    const { getByRole } = render(<CardOverview slug={'my-journey'} />)
-    expect(getByRole('link')).toBeInTheDocument()
-    expect(getByRole('link')).toHaveAttribute(
-      'href',
-      '/journeys/my-journey/edit'
+  it('should render description for no cards', () => {
+    const { getByText } = render(<CardOverview slug={'my-journey'} />)
+    expect(getByText('No cards in this journey')).toBeInTheDocument()
+  })
+  it('should have edit button', () => {
+    const { getByRole } = renderWithApolloClient(
+      <CardOverview slug={'my-journey'} blocks={steps} />
+    )
+    expect(getByRole('link', { name: 'Edit' })).toBeInTheDocument()
+  })
+  it('should have text on how many cards present in journey', () => {
```

Comment:
```suggestion
  it('should render description for multiple cards', () => {
```
Just trying to keep the wording consistent